### ISSUE TITLE
Conflict between SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing and Squiz.WhiteSpace.ControlStructureSpacing

### ISSUE THREAD

**Issue by morozov** [slevomat/coding-standard#1086]
```
$ composer show | grep -P 'coding-standard|php_codesniffer'

doctrine/coding-standard                       8.1.0  The Doctrine Coding Standard is a s...
slevomat/coding-standard                       6.4.0  Slevomat Coding Standard for PHP_Co...
squizlabs/php_codesniffer                      3.5.6  PHP_CodeSniffer tokenizes PHP, Java...
```
Consider the following code example:

```php
<?php

function test()
{
    switch (true) {
        default:
            switch (true) {
                default:
                    break;
            }
    }
}

```
Try checking:
```
$ phpcs \
    -s \
    --standard=Doctrine \
    --sniffs=SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing,Squiz.WhiteSpace.ControlStructureSpacing \
    test.php

FILE: /home/morozov/Projects/phpcs-playground/test.php
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 6 | ERROR | [x] Expected 1 lines after "default", found 0.
   |       |     (SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------

Time: 66ms; Memory: 10MB
```
Try fo fix:
```
$ phpcbf \
    --standard=Doctrine \
    --sniffs=SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing,Squiz.WhiteSpace.ControlStructureSpacing \
    test.php

PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
/home/morozov/Projects/phpcs-playground/test.php      FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 143ms; Memory: 10MB
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/ee75ad84cdef36c6382e787fc0f3c41a051a0621
